boto/ec2/elb/__init__.py

```diff
@@ -756,3 +757,71 @@ def detach_lb_from_subnets(self, name, subnets):
                                'Subnets.member.%d')
         return self.get_list('DetachLoadBalancerFromSubnets',
                              params, None)
+
+    # Tag methods
+
+    def _build_tag_param_list(self, params, tags):
```

Comment:
We can improve a bit this method:
- make it class method or static method as it doesn't depend on the class state
- avoid side-effects - not pass `params` for the modification but return a new instance on every call
- rename to not confuse - we expect getting `list` as a result, but we get mapping. 

```
@classmethod
def _build_tag_parameters(cls, tags):
    params = {}
    for idx, key in enumerate(sorted(tags), start=1):
        params['....'] = key
        value = tags[key]
        if value is not None:
            params['...'] = value
    return params        
        
```